azure/storage.sh

```diff
@@ -1,14 +1,15 @@
 # Storage related functions
 
 fn azure_storage_account_create(name, group, location, sku, kind) {
```

Comment:
And, this is the first function of `azure` library that returns a resource id. In every other function the resource id is managed by user.